assets/js/editor-components/incompatible-payment-gateways-notice/index.tsx

```diff
@@ -85,7 +85,7 @@ export function IncompatiblePaymentGatewaysNotice( {
 									key={ id }
 									className="wc-blocks-incompatible-extensions-notice__element"
 								>
-									{ title }
+									• { title }
```

Comment:
Shouldn't we use list styling instead of a dot? don't feel like you need to change this, just asking.

Reply:
By changing the display from the default `display: list-item` to `flex`, some properties of lists are lost (e.g., `list-style-type`). But, we can use the `before` CSS property instead! 🙌 I pushed the changes